pro_1.py

```diff
@@ -0,0 +1,92 @@
+import json
+from re import A
+import re
+
+database_file=open("database.txt","r+")
+dict_n=json.load(database_file)
+def loged_in():
+    print("succes")
+
+def login():
+    print("login page")
+    return
+def signup():
+    pattern1="\w{1,}.\w{1,}@\w{1,}.com"
+    pattern2="^.*(?=.{8,})(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=]).*$"
+    print("signup page")
+    name=input("enter ur full name ")
```

Comment:
Try to enhance the experience a little bit for the user to interact with the screen.

```suggestion
    print("======= SIgn up page =======")
    name=input("[Sign up step 1/4]: Enter your full name: ")
```